api/src/Entity/MaterialList.php

```diff
@@ -34,7 +34,9 @@
             security: 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)'
         ),
         new Delete(
-            security: 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)'
+            security: 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)',
+            validate: true,
+            validationContext: ['groups' => ['delete']]
```

Comment:
Where is this group set?

Reply:
We use it a bit further below in Line 62 in the attribute for the validation constraint:
https://github.com/ecamp/ecamp3/pull/5275/files#diff-4e6f5c93038c615b8c2dc034a317af9338b08371dda797697fff316b0b9a819fR62

See also the example in the api-platform docs:
https://api-platform.com/docs/core/validation/#using-validation-groups-on-operations

Or specifically on delete operation:
https://api-platform.com/docs/guides/delete-operation-with-validation/